plugins/org.jboss.reddeer.workbench/src/org/jboss/reddeer/workbench/impl/editor/AbstractEditor.java

```diff
@@ -292,4 +333,40 @@ public ContentAssistant getAutoContentAssistant(Runnable execute){
     public IEditorPart getEditorPart() {
         return editorPart;
     }
+    
+    public enum ContentAssistantEnum{
+    	DEFAULT("Default"),
```

Comment:
some of these enums are specific to JBossTools. We should list only those who are available in Eclipse JEE distribution